test/tools/ossfuzz/config/solidity.dict

```diff
@@ -40,16 +41,20 @@
 ".delegatecall("
 ".gas("
 ".kill("
-".shutdown("
 ".length"
 ".pop();"
 ".push("
 ".runtimeCode"
 ".send("
+".shutdown("
 ".staticcall("
 ".transfer("
 ".value"
 "/"
+"// compileViaYul: false"
+"// compileViaYul: true"
+"// optimize: false"
+"// optimize: true"
```

Comment:
If a file already contains `// optimize: false`, is it guaranteed that the mutation would after ``// optimize: false`?

Reply:
Sadly not, the mutations are probabilistic, so duplicates are possible. However, given a high execution throughput (in the order of tens of tests per second), eventually the interesting inputs get picked up and amplified (via code coverage).